client/ayon_core/hosts/unreal/api/hierarchy.py

```diff
@@ -0,0 +1,79 @@
+import os
+
+from ayon_api import get_folders_hierarchy
+from ayon_core.settings import get_project_settings
+from ayon_core.hosts.unreal.api.pipeline import (
+    generate_sequence,
+    set_sequence_hierarchy,
+)
+
+
+def get_default_sequence_path(settings):
+    """Get default render folder from blender settings."""
+
+    sequence_path = settings['unreal']['sequence_path']
+    if sequence_path.endswith("/"):
+        sequence_path = sequence_path.rstrip("/")
+
+    return f"/Game/{sequence_path}"
+
+
+def _create_sequence(
+        element, sequence_path, parent_path="", parent_seq=None,
+        parent_fr=None
```

Comment:
Can we use just the full argument names for these, `parent_frame_range` is much clearer?